.gitignore

```diff
@@ -10,7 +10,7 @@
 .svn/
 .vscode/
 pubspec.lock
-!example/pubspec.lock
```

Comment:
I didn't know we could do it 👀.  I thought it was required by pub.dev

Reply:
nope it's not required by pub.dev, and it's "best practice" to version control it in the final application, not in packages